src/NetSparkle/SparkleUpdater.cs

```diff
@@ -96,10 +97,12 @@ public SparkleUpdater(string appcastUrl, ISignatureVerifier signatureVerifier, s
         /// <param name="signatureVerifier">the object that will verify your app cast signatures.</param>
         /// <param name="referenceAssembly">the name of the assembly to use for comparison when checking update versions</param>
         /// <param name="factory">a UI factory to use in place of the default UI</param>
-        public SparkleUpdater(string appcastUrl, ISignatureVerifier signatureVerifier, string referenceAssembly, IUIFactory factory)
+        /// <param name="matcher">The optional matching delegate that can be used to filter appcastitem elements in/out of the upgrade process</param>
+        public SparkleUpdater(string appcastUrl, ISignatureVerifier signatureVerifier, string referenceAssembly, IUIFactory factory, IAppCastFilter matcher)
```

Comment:
In order to not break previous uses, need to have `IAppCastFilter matcher = null` at the end, I believe?

Reply:
Fixed. 

I have removed the IAppCastFilter entirely from the IAppCastHandler as well - and while this protects everyone against an interface change (yay); it means there is no nice way to inject the IAppCastMatcher (boo).  Currently, I'm instantiating the SparkleUpdater and then *manually* injecting it - an example of which is in the SparkleUpdaterFixture class, within the CreateUpdater method. 

Personally; I find this to be awful - as it requires internal knowledge of both the SparkleUpdater and XmlAppCast objects - the only benefit is that doing it this way is opt in, and avoids any change to the existing interface signatures. 

I am really not a fan of the current method to inject the filter; perhaps you see a way to inject the filter into the process that I'm currently missing/blind to?